install-cloud-sdk-build-task/install-cloud-sdk.ps1

```diff
@@ -43,7 +43,7 @@ try {
             gcloud config set disable_usage_reporting $disableReportingString
         }
     } else {
-        $InstallerUri = "https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.zip"
+        $InstallerUri = "https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk-windows-bundled-python.zip"
```

Comment:
Is this URL guaranteed to always be the latest bits? Should we fix it at a know version instead? This URL does not show up in the [versioned](https://cloud.google.com/sdk/downloads#versioned) download page, we should probably use one of those instead.